### ISSUE TITLE
There's no data for the selected runs. Try a different X axis setting.

### ISSUE THREAD

**Issue by ninastijepovic** [wandb/client#1022]
`wandb, version 0.8.17 && Python 3.7.6 && uname`

* Weights and Biases version:version 0.8.17 
* Python version:Python 3.7.6
* Operating System:CentOS

### Description
Hello,
I have been trying to add plots to my wandb project, however, I have been receiving the following error:
There's no data for the selected runs.
Try a different X axis setting.
Current X axis: _step

### What I Did
Here is the plotting function
```
#plot analysis
def plot_sample(X, y, preds, binary_preds, ix=None):
    if ix is None:
        ix = random.randint(0, len(X))

    has_mask = y[ix].max() > 0

    fig, ax = plt.subplots(1, 4, figsize=(20, 10))
    ax[0].imshow(X[ix, ..., 0], cmap='seismic')
    if has_mask:
        ax[0].contour(y[ix, ..., 0].squeeze(),  levels=[0.5])
    ax[0].set_title('Signal')

    ax[1].imshow(y[ix, ..., 0].squeeze())
    ax[1].set_title('Mask')

    ax[2].imshow(preds[ix, ..., 0].squeeze(), vmin=0, vmax=1)
    if has_mask:
        ax[2].contour(y[ix, ..., 0].squeeze(),  levels=[0.5])
    ax[2].set_title('Mask predicted')
    ax[3].imshow(binary_preds[ix, ..., 0].squeeze(), vmin=0, vmax=1)
    if has_mask:
        ax[3].contour(y[ix, ..., 0].squeeze(), levels=[0.5])
    ax[3].set_title('Signal Predicted binary')

wandb.log({'Analysis':plot_sample(processed_data,masks,preds,preds_data)})
```
What could be the issue? I am able to get the plots but not in the wandb project. 
Thank you for the help!

**Comment by cvphelps** [comment]
Hi @ninastijepovic thanks for reporting this. Could you share a link to the project where you're seeing this issue with nothing showing up? It looks like you're logging masks to overlay with images. @nbardy could you follow up

**Comment by nbardy** [comment]
Hey @ninastijepovic it looks like you are using Plotly or possibly matplotlib to log image masks. We recently released support for images masks natively which contains more features and requires less setup to log.

You can try it out here:
https://docs.wandb.com/library/log#images-and-overlays

**Comment by ariG23498** [comment]
Hey @ninastijepovic 
In the past year we've majorly reworked the CLI and UI for Weights & Biases. We're closing issues older than 6 months. Please comment to reopen.

**Comment by HimGautam** [comment]
Hi,
I am having the same issue. I am using ```wandb.log()``` to log the data generated by reinforcement learning algorithm. On the webpage, it shows some all the data except for the one I am logging after several timesteps. The code used for logging is shown below:

```
best_score = -100000
score_history = deque(maxlen=100)
global_step = 0
for ep in range(0, num_episodes):
    state = env.reset()
    done = False
    score = 0
    for step in range(num_steps):
        if global_step<start_after:
            action = env.sample_action()
        else:
            action =  agent.act(np.expand_dims(state, axis = 0), noise_std = noise_std)
            action = np.squeeze(action)
            
        next_storage_state, next_state, reward, done =  env.step(action, global_step)    
        score += reward
        global_step += 1
        agent.memory.remember(state, action, reward, next_storage_state, done)
        state = next_state
        if global_step>update_after:
            actor_loss, critic_loss = agent.learn()
            
            wandb.log({"Actor Loss": actor_loss})
            wandb.log({"Critic Loss": critic_loss})
        
    env.next_ep() 
    wandb.log({"Score": score})
    print(f"Episode: {ep}, Len: {step}, Score: {score}, Global_Step: {global_step}, Done: {done}")  
 ```
![Screenshot from 2022-08-25 13-23-09](https://user-images.githubusercontent.com/70597091/186607936-3a11aad7-1ba5-4f12-b330-8dee827bcca8.png)

It is showing the ```actor_loss``` and ```critic_loss``` but the ```score```  is not there.  
Here, num_steps=9,000.
